client/src/utils/slate/handleMovingCursor.ts

```diff
@@ -0,0 +1,26 @@
+import { Change, Value } from "../../components/BlockEditor/types";
+
+export function handleMovingCursor(change: Change, value: Value, isMovingLeft: boolean, isMovingRight: boolean) {
```

Comment:
If this function has 4 arguments, I'd prefer that it have a hash input or similar. Typically I prefer hash inputs for anything over 2 args.